### PR TITLE
Add is_valid_params

### PR DESCRIPTION
**Pull request by nrryuya**

### CODE REVIEW

File: CBCCasper.thy [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+(* Section 2.1: CBC Casper "Parameters" *)
+
+notation Set.empty ("\<emptyset>")
+
+definition is_non_empty :: "'a set \<Rightarrow> bool"
+  where
+    "is_non_empty bs = ( \<exists> b. b \<in> bs )"
+
+(* Definition 2.1 ~ 2.5 *)
+datatype validator = Validator int
```
Comment (yudetamago) [inline CBCCasper.thy]: `Validator real` ?

Comment (nrryuya) [reply CBCCasper.thy]: The data type of `validator names` is not defined in the paper. I prefer `int` to represent IDs of validators.

**Comment by nrryuya** [comment]
>btw you can check whether or not errors are contained by using the command like this 

Cool! It's in #24

**Comment by nrryuya** [comment]
>Cool! It's in #24

hmm... It's not.

**Pull request merged**